hudi-client/hudi-client-common/src/test/java/org/apache/hudi/utils/TestMetadataConversionUtils.java

```diff
@@ -82,6 +82,7 @@ public void init() throws IOException {
   public void testCompletedClean() throws Exception {
     String newCommitTime = HoodieTestTable.makeNewCommitTime();
     createCleanMetadata(newCommitTime);
+    metaClient.reloadActiveTimeline();
     HoodieArchivedMetaEntry metaEntry = MetadataConversionUtils.createMetaWrapper(
```

Comment:
Do we still need the reload?

Reply:
Yes, this is necessary, MetadataConversionUtils.createMetaWrapper will try to get the complete instant path from activeTimeline, but if we don't reloadActiveTimeline, the complete instant written above "createCleanMetadata(newCommitTime);" will not be included in activeTimeline.

Reply:
Remove other reloadActiveTimeline except `testCompletedCompaction`, since it will initialize activeTimeline when create commits. We have to reload here to ensure new commits are added.